app/Models/Router.php

```diff
@@ -53,6 +53,13 @@ public function sendWarning()
             foreach ($internet_admins as $admin) {
```

Comment:
Also please do https://github.com/EotvosCollegium/mars/issues/70 also, while you are here